### DISCUSSION TITLE
my minio systemd can't start

### DISCUSSION THREAD

**Discussion by FrandeScarlet** [minio/minio#17272] [Q&A] [answered]
i get this error after i start the systemd service
systemctl start minio.service
![image](https://github.com/minio/minio/assets/51735373/eb84d1af-d821-4d72-a55c-d41751cbaef9)
anyone can help me?

**Comment by vadmeste** [comment, marked as answer]
@FrandeScarlet 'journalctl -u minio.service' will probably show the error that makes MinIO service to exit early during boot up

**Comment by FrandeScarlet** [comment]
![image](https://github.com/minio/minio/assets/51735373/fa4094b4-2ada-46e8-8075-c85b7e64ed19)

![image](https://github.com/minio/minio/assets/51735373/7f338a9f-863f-4011-a9ee-0b815a0edd53)

**Comment by harshavardhana** [comment]
You have create the user first that you want the service to run with.

**Reply by FrandeScarlet** [reply]
Can you please show the example how to create the user? In where?
